Jenkinsfile

```diff
@@ -26,14 +34,23 @@ elifeLibrary {
             elifeGitMoveToBranch commit, 'master'
         }
 
-        stage 'Push image', {
+        stage 'Push unstable image', {
             def image = DockerImage.elifesciences(this, 'sciencebeam', commit)
+            image = image.addSuffixAndTag('_unstable', commit)
             image.push()
             image.tag('latest').push()
         }
 
+        stage 'Push release image', {
+            isNew = sh(script: "git tag | grep v${candidateVersion}", returnStatus: true) != 0
+            if (isNew) {
+                def image = DockerImage.elifesciences(this, 'sciencebeam', commit)
+                image.tag(candidateVersion).push()
+            }
+        }
+
         stage 'Downstream', {
-            build job: '/dependencies/dependencies-sciencebeam-texture-update-sciencebeam', wait: false, parameters: [string(name: 'commit', value: commit)]
+            build job: '/dependencies/dependencies-sciencebeam-texture-update-sciencebeam', wait: false, parameters: [string(name: 'commit', value: candidateVersion)]
```

Comment:
passing down release version as `commit` which should work the same as it's really just the tag. Will rename parameter.

Reply:
Yes, I agree if there are releases available this should be wrapped with `if (isNew)`